### PR TITLE
ROC-2855: Update frontend to match typescript

### PR DESCRIPTION
**Pull request by jacmurphy**
### Jira link
https://tools.hmcts.net/jira/browse/ROC-2855

### Updates in other projects 
https://github.com/hmcts/cmc-citizen-frontend/pull/291

### Description 
Flattens out the structure of `if` statements and makes it more maintainable in the future

### CODE REVIEW

File: macros/responseStatus.njk [hunk 1]
```diff
@@ -7,33 +7,19 @@
 
 {% macro responseStatusForDefendant(claim, url, isAfter4pm) %}
 
-  {% if claim.settlementReachedAt  %}
+  {% if claim.settlementReachedAt %}
```
Comment (timja) [inline macros/responseStatus.njk]: shouldn't this also be a status?

Comment (jacmurphy) [reply macros/responseStatus.njk]: Yes it should, Updating now

File: macros/responseStatus.njk [hunk 2]
```diff
@@ -6,34 +6,21 @@
 {% endmacro %}
 
 {% macro responseStatusForDefendant(claim, url, isAfter4pm) %}
+  {{ claim.status }}
```
Comment (timja) [inline macros/responseStatus.njk]: hmm

Comment (jacmurphy) [reply macros/responseStatus.njk]: 😳

**Review by timja — APPROVED** [review]
I'll erm trust you've tested all these 👍

**Pull request merged**